libraries/node-core-library/src/test/Text.test.ts

```diff
@@ -3,8 +3,8 @@
 
 import { Text } from '../Text';
 
-describe(Text.name, () => {
-  describe(Text.padEnd.name, () => {
+describe('Text', () => {
+  describe('padEnd', () => {
```

Comment:
Unless `Text.padEnd.name` no longer resolves at runtime, prefer that form for the naming so that the test titles stay synchronized if the functions are renamed.